### PR TITLE
feat: added version text to Splash screen

### PR DESCRIPTION
**Pull request by DimitrySakalouski**

### CODE REVIEW

**Comment by DimitrySakalouski** [comment]
image

**Comment by DimitrySakalouski** [comment]
image

File: src/shared/ui/Splash.tsx [hunk 1]
```diff
@@ -16,6 +24,8 @@ const SplashScreen = ({ isAppReady }: { isAppReady: boolean }) => {
   const containerOpacity = useRef(new Animated.Value(1)).current;
   const imageOpacity = useRef(new Animated.Value(0)).current;
 
+  const appVersion = DeviceInfo.getVersion();
```
Comment (BamMironov) [inline src/shared/ui/Splash.tsx:27]: Please use this line instead:
`const appVersion = ENV ? `${APP_VERSION} ${ENV}` : APP_VERSION,`

from:
image

**Review by BamMironov — CHANGES_REQUESTED** [review]

**Comment by DimitrySakalouski** [comment]
image

**Comment by DimitrySakalouski** [comment]
image

**Review by BamMironov — APPROVED** [review]

**Review by BamMironov — APPROVED** [review]

**Pull request merged**